packs/spell-effects/spell-effect-lift-natures-caul-bonus.json

```
@@ -0,0 +1,44 @@
+{
+    "_id": "ihv1azg80N3kj7Vo",
+    "img": "systems/pf2e/icons/spells/lift-natures-caul.webp",
+    "name": "Spell Effect: Lift Nature's Caul (Bonus)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by <em>@UUID[Compendium.pf2e.spells-srd.Item.Lift Nature's Caul]</em></p>\n<p>Aberrations or worshippers of Outer Gods or Great Old Ones are bolstered by this vision of warped reality and gain a +1 status bonus to Will saving throws and attack rolls for 1 minute.</p>"
```

```suggestion
            "value": "<p>Granted by <em>@UUID[Compendium.pf2e.spells-srd.Item.Lift Nature's Caul]</em></p>\n<p>The creature gains a +1 status bonus to Will saving throws and attack rolls.</p>"
```
Effects should say what they do and minimize other text unless it's useful for people to reference. Duration info is in the sidebar already.